src/components/SearchApp/SearchApp.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+import classes from './Container.css'
+import Search from '../Search/Search'
+import Table from '../Table/Table'
+import CheckPrevious from '../CheckPrevious/CheckPrevious'
+
+const SearchApp = (props) => {
+  const [persons, setPersons] = useState(props.persons)
+  const [name, setName] = useState('')
+  const [pastSearches, setPastSearches] = useState([])
+
+  const handleChangeName = (event) => {
+    setName(event.target.value)
+  }
+
+  const getPersons = (name) => {
+    return props.persons.filter((person) => {
+      if (person.name.toLowerCase().startsWith(name)) {
+        return person
+      }
+      return null
```

Comment:
return empty array

Reply:
this filter returns a person object if condition true so if false it will return null
```suggestion
      return null
```